GiTracker/Services/Issues/IssueService.cs

```diff
@@ -23,6 +23,7 @@ public IssueService(IRestService restService, IGitApiProvider gitApiProvider)
             var issues =
                 await
                     _restService.GetAsync(_gitApiProvider.Host, _gitApiProvider.GetIssuesUrl(repository),
+                        new Dictionary<string, string> {{"state", "all"}},
```

Comment:
GitLab may not have this exact parameter. 
Creating parameters Dictionary should sit in _gitApiProvider